### ISSUE TITLE
ERA_Collections: Official collection of collections

### ISSUE THREAD

**Issue by pbinkley** [ualbertalib/HydraNorth#76]
As admin I want to be able to select which collections are ‘official’ and displayed on the main ERA/repository ‘front door’, and to be able to refer to these collections with permanent ids
Implement as new tab, pending Natasha's recommendation
Admin can create communities and collections, and assign their nesting relationships
Filter based on a flag for inclusion in public navigational structure
(Note: will have to revisit when Hydra::Works comes out)

- [ ] implementing collections#index route so that there is a target which enumerates collections
- [ ] admin should be able to make collections official from collections#show
- [ ] only official collections appear in collections#index 
- [ ] admin should be able to remove official collections from collections#index
- [ ] add communities tab to homepage which links to collections#index
- [ ] link "Collections" browse links on front page to new collections#index (from #248)
- [ ] collections#show and collections#index should include share-with-me links
- [ ] when browsing collections, we want to go to the collection home page rather than to a facet query for items belonging to that collection
- [ ] allow users to browse by communities, not collections (since collection titles are not sufficiently descriptive) - depends on #569

**Comment by pbinkley** [comment]
Question: will collections have ARKs? Existing handles are well used.

**Comment by pbinkley** [comment]
May depend on Hydra::Works.

**Comment by weiweishi** [comment]
Collection model in Sufia includes: Sufia::Noid. But no ARK yet.

**Comment by pgwillia** [comment]
note: [FeaturedWorks](https://github.com/projecthydra/sufia/blob/master/sufia-models/app/models/featured_work.rb) is limited 5 and stored in a db table. Not exactly the model I want.

Want to be able to facet based on featured => indexed in Solr

Propose 
- [ ] implementing collections#index route so that there is a target which enumerates collections
- [ ] admin should be able to feature collection from collections#show
- [ ] only featured collections appear in collections#index 
- [ ] admin should be able to un-feature collection from collections#index
- [ ] add communities tab to homepage which links to collections#index
- [ ] results should include share-with-me links

Won't do
 * assigning nested relationships #75

**Comment by pbinkley** [comment]
This looks good. I think it would be good to use a different name than
"featured": maybe "published". "Featured" suggests items selected out of
the mass for special attention, whereas our feature is intended to give the
*full* set of public collections that are part of the "official" HydraNorth
navigation.

That would allow it to live alongside a "featured" collection feature in
the narrow sense, too.

Peter

**Comment by pgwillia** [comment]
How about curate and weed instead of publish/un-publish, feature/un-feature or official/unofficial?

**Comment by pbinkley** [comment]
hmm - curate could more precisely apply to a lot of things that we might
ultimately want to do with collections (e.g. allow trusted user to manage
an "official" collection, to give it intellectual authority). And we might
want the "feature" feature (sigh) along side the official navigation
structure.

The essence of it is to provide a navigational structure designed and
maintained by ERA admin, in contrast to any user-created collections that
might arise. But we know that ERA admin might also have collections that
don't belong in this structure (but which might still be public, so the
"publish" label might be confusing). Maybe "bless"/"unbless"?

Peter

**Comment by gsharder** [comment]
Official/unofficial makes sense to me -- my two cents.

On Tue, Mar 3, 2015 at 3:43 PM, Peter Binkley <notifications@github.com>
wrote:

> hmm - curate could more precisely apply to a lot of things that we might
> ultimately want to do with collections (e.g. allow trusted user to manage
> an "official" collection, to give it intellectual authority). And we might
> want the "feature" feature (sigh) along side the official navigation
> structure.
>
> The essence of it is to provide a navigational structure designed and
> maintained by ERA admin, in contrast to any user-created collections that
> might arise. But we know that ERA admin might also have collections that
> don't belong in this structure (but which might still be public, so the
> "publish" label might be confusing). Maybe "bless"/"unbless"?
>
> Peter
>
>
>
> On 3 March 2015 at 15:28, pgwillia <notifications@github.com> wrote:
>
> > How about curate and weed instead of publish/un-publish,
> > feature/un-feature or official/unofficial?
> >
> > —
> > Reply to this email directly or view it on GitHub
> > <
> https://github.com/ualbertalib/HydraNorth/issues/76#issuecomment-77050808>
>
> > .
> >
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/ualbertalib/HydraNorth/issues/76#issuecomment-77054510>
> .
>



-- 
Geoff Harder
Associate University Librarian
Information Resources and Digital Initiatives

*University of Alberta Libraries*


*2-10H Cameron Library, University of AlbertaEdmonton, AB, Canada T6G 2J8
 780.492.7914  *
*geoffrey.harder@ualberta.ca <geoffrey.harder@ualberta.ca> *
*library.ualberta.ca <http://library.ualberta.ca>*

**Comment by sfarnel** [comment]
I think official/unofficial is simple and clear.

On Tue, Mar 3, 2015 at 4:41 PM, gsharder <notifications@github.com> wrote:

> Official/unofficial makes sense to me -- my two cents.
>
> On Tue, Mar 3, 2015 at 3:43 PM, Peter Binkley <notifications@github.com>
> wrote:
>
> > hmm - curate could more precisely apply to a lot of things that we might
> > ultimately want to do with collections (e.g. allow trusted user to manage
> > an "official" collection, to give it intellectual authority). And we
> might
> > want the "feature" feature (sigh) along side the official navigation
> > structure.
> >
> > The essence of it is to provide a navigational structure designed and
> > maintained by ERA admin, in contrast to any user-created collections that
> > might arise. But we know that ERA admin might also have collections that
> > don't belong in this structure (but which might still be public, so the
> > "publish" label might be confusing). Maybe "bless"/"unbless"?
> >
> > Peter
> >
> >
> >
> > On 3 March 2015 at 15:28, pgwillia <notifications@github.com> wrote:
> >
> > > How about curate and weed instead of publish/un-publish,
> > > feature/un-feature or official/unofficial?
> > >
> > > —
> > > Reply to this email directly or view it on GitHub
> > > <
> >
> https://github.com/ualbertalib/HydraNorth/issues/76#issuecomment-77050808>
> >
> > > .
> > >
> >
> > —
> > Reply to this email directly or view it on GitHub
> > <
> https://github.com/ualbertalib/HydraNorth/issues/76#issuecomment-77054510>
> > .
> >
>
>
>
> --
> Geoff Harder
> Associate University Librarian
> Information Resources and Digital Initiatives
>
> *University of Alberta Libraries*
>
>
> *2-10H Cameron Library, University of AlbertaEdmonton, AB, Canada T6G 2J8
> 780.492.7914 *
> *geoffrey.harder@ualberta.ca <geoffrey.harder@ualberta.ca> *
> *library.ualberta.ca <http://library.ualberta.ca>*
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/ualbertalib/HydraNorth/issues/76#issuecomment-77064254>
> .
>



-- 
Sharon Farnel
Metadata Coordinator
University of Alberta Libraries
sharon.farnel@ualberta.ca
780-492-3685

**Comment by pgwillia** [comment]
- [ ] implementing collections#index route so that there is a target which enumerates collections

[Fixed upstream](https://github.com/projecthydra-labs/hydra-collections/issues/58)

**Comment by pgwillia** [comment]
This issue changed significantly with the decision to remove admin functionality from the front page.  My understanding of how this will be handled is that links to desired "official" collections will be hard coded into the front page of the application.  The existing collections display page functionality plus the ability to nest collections is sufficient for collection browsing. The batch ingest/migration of collections will provide the content to link to.

I think this issue can be closed as a result of these decisions.

**Comment by pbinkley** [comment]
I agree - we'll just need to coordinate with Natasha on who will implement this functionality as part of the structure of second-level browse pages.

**Comment by pbinkley** [comment]
We need to check with Natasha - we think that all that's needed is styling work from Natasha. @nnunn , please check with Chris about this.

**Comment by pbinkley** [comment]
Sonya will flesh out the presentation requirements for this functionality.

**Comment by nnunn** [comment]
Community page is now up and working.
